src/components/sections/Catalogue/CurrentProductSection/components/Articles/ProductDescription/ProductDescription.tsx

```
@@ -1,36 +1,74 @@
+import RadioBtnGroup from './components/RadioBtnGroup'
+import ReviewsBlock from './components/ReviewsBlock'
+import SizeSelector from './components/SizeSelector'
+
+export default function ProductDescription({
+  product,
+  reviews,
+  isReviewsLoading,
+  isReviewsError
+}: {
+  product?: Product
+  reviews?: any[]
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)